New scenarios: KotH, Shattered Alliance, Astral Run (#211)
qwrtln/Homm3BG-mission-book: draft-scenarios/clash/king_of_the_hill_v2.tex

\begin{multicols}{2}

\textbf{Author:} KABLAMIO \& Cantila

\textbf{Source:} \href{https://discord.com/channels/1275490124301467799/1284428969369665631}{KABLAMIO's Discord}

\textit{All eyes turn to the Hill — where legends rise and empires fall. Fight, claim it, and prove you’re worthy to rule.}

\subsection*{\MakeUppercase{Scenario Length}}
This Scenario plays out over 6 Rounds.

\subsection*{\MakeUppercase{Player Setup}}
\textbf{Player Count:} 2 -- 3

\textbf{Starting Resources:} 22 \svg{gold}, 5 \svg{building_materials}

\textbf{Starting Income:} 15 \svg{gold}, 2 \svg{building_materials}, 1 \svg{valuables}

\textbf{Starting Units:}
\begin{itemize}
  \item A Few of cheapest \silver\ Units
\end{itemize}

\textbf{Town Buildings:}
\begin{itemize}
  \item \bronze\ Dwelling
\end{itemize}

\subsection*{\MakeUppercase{Map Setup}}
Take the following Map Tiles and arrange them as shown in the Scenario map layout ($P$ stands for the number of players):

\begin{itemize}
  \item P × Starting (I) Map Tile
  \item P × Far (II-III) Map Tiles
  \item P × Near (IV-V) Map Tiles
  \item Center Map Tile (C1 or C3)

>   \item Custom Center Map Tile

Reviewer: I wonder if it needs any additional explanation, even something as simple as "see map layout", as there are no other scenarios with custom tiles, at least in this book.

\end{itemize}

\subsection*{\MakeUppercase{Victory Points}}

> After Round 6, the Player with the most Victory Points (VP) wins. Players get
> VPs for:

Reviewer: ```suggestion
\subsection*{\MakeUppercase{Victory Points}}
After Round 6, the player with the most Victory Points (VP) wins.
Players get VPs for:
```

\begin{itemize}

>   \item Round number - 1 VP for starting your turn on ``The Hill''

Reviewer: ```suggestion
  \item 1 VP for starting your turn on ``The Hill''
```
I'm not sure if I get this round number?

Author: on round 2, you get 1 VP, round 3 = 2VPs, Round 4 =3VPs and so on

Author: X -1 VPs (where X is the Round number)

Reviewer: Ohh, then it needs a math mode, hold on.

>   \item Round number - 1 VP for starting your turn on ``The Hill''

Reviewer: ```suggestion
  \item $R-1$ VPs for starting your turn on ``The Hill'', where $R$ is the round number.
```
Something like that maybe?

Author: with capitalized Round or round?

Reviewer: Capitalized: Round, sorry.

  \item 2 VP for winning a Combat against another Player
  \item 1 VP for losing a Combat against another Player
\end{itemize}

\subsection*{\MakeUppercase{Timed Events}}

\begin{itemize}

>   \item \textbf{\nth{2} Round:} Gain secondary Hero for free.

Reviewer: ```suggestion
  \item \textbf{\nth{2} Round:} Gain Secondary Hero for free.
```

>   \item \textbf{\nth{3} Round:} Gain Citadel for free. If you had it, gain 1 VP.

Reviewer: ```suggestion
  \item \textbf{\nth{3} Round:} Gain Citadel for free. If you already had it, gain 1 VP.
```

>   \item \textbf{\nth{4} Round:} Gain \silver\ dwelling for free. If you had it: gain 2 VP.

Reviewer: ```suggestion
  \item \textbf{\nth{4} Round:} Gain \silver\ dwelling for free. If you already had it, gain 2 VP.
```

>   \item \textbf{\nth{5} Round:} Gain \golden\ dwelling for free. If you had it: gain 3 VP.

Reviewer: ```suggestion
  \item \textbf{\nth{5} Round:} Gain \golden\ dwelling for free. If you already had it, gain 3 VP.
```

\end{itemize}

\subsection*{\MakeUppercase{Additional Rules}}
\begin{itemize}

>   \item Every Round after Round 1 is a Resource Round.

Author: hmm, yeah I could change that to "Gain income also on even-numbered Rounds"

>   \item Every Round after Round 1 is a Resource Round.

Reviewer: Or simply something like "There are no Astrologers Rounds."

  \item Players can't attack their opponent's starting Town.

>   \item Players can only attack another player once each turn.

Reviewer: You get only one attack per turn, or you can attack two players?

Author: I will reword this to "A player cannot attack the same player twice in one Round."

Author: or "Players cannot..."

Reviewer: How about three times? 😆 Let's make it "more than once".

Author: as a mathematician I will counter that to attack three times you have to have already broken the rule to not attack twice :D but yeah more than once is better (y)

  \item Players do not lose gold for losing or retreating from any Combat.

>   \item Secondary Heroes act as Main Hero.

Reviewer: ```suggestion
  \item Secondary Heroes act as Main Heroes.
```
What does it mean exactly? You get to use your deck? They get their own hero board, stargin ability, etc.?

Author: Secondary hero is a clone of your main hero / your main hero is in two places at the same time. Same heroboard, same deck of cards.

Reviewer: OK, then I'd say it needs some more explaining. It's virtually the same hero in a different place, but with 2 MPs, right?

>   \item After losing a Combat, the defeated Hero is moved to it's starting Town.

Reviewer: ```suggestion
  \item After losing a Combat, the defeated Hero is moved to its starting Town.
```

  \item After PvP Combat, both Players gain their lost \bronze\ and \silver\ Units back.
  \item Players may \svg{pay_v2} 6 \svg{gold} and 6 \svg{building_materials} to gain Walls and Arrow Tower until their next turn.

>   \item Level VI and VII Combats cannot be skipped.
>   \item Level VII Neutral Combat reward: Gain 20 \svg{gold}.

Reviewer: ```suggestion
  \item Level VI and VII Combats cannot be skipped.
  \item ``The Hill'' Combat reward: 20 \svg{gold}.
```
The Hill is the only VII field, right? In that case, it could be explicit.

>   \item \textbf{Obelisks:} Search (2) \svg{artifact}. \textit{Visitabe once per Faction.}

Reviewer: ```suggestion
  \item \textbf{Obelisks:} Search (2) \svg{artifact}. \textit{Visitable once per Faction.}
```

>   \item \textbf{Dragon Utopias:} On Center Map Tile a Hero can spend 1 \svgeven{movement} to teleport to Dragon Utopia. After Flagging Dragon Utopia, gain 10\svg{gold}, resolve all rewards from adjacent Fields. You may also recruit one defeated Neutral Unit for free.

Reviewer: ```suggestion
  \item \textbf{Dragon Utopias:} On Center Map Tile a Hero can spend 1 \svgeven{movement} to teleport to Dragon Utopia. After Flagging Dragon Utopia, gain 10 \svg{gold}, resolve all rewards from adjacent Fields. You may also recruit one defeated Neutral Unit for free.
```
OK, if there's a dragon utopia, then hill is not the only VII field 🤔 But I guess the 20 gold reward does not cover it?

Author: removed the 10 gold gain.

Reviewer: Wait, so you still get 20 gold for beating dragon utopia?

\end{itemize}

\subsection*{\MakeUppercase{Optional Rules}}
\begin{itemize}
  \item After winning a Neutral Combat, you may Recruit one of the defeated Neutral Units for half the recruitment \svg{gold} rounded up (you may use only one Neutral Unit Recruited this way in a Combat).  % no-check-caps

>   \item When a Unit with \svg{defense} is attacked but suffers no \svg{damage-red}, it gains a \href{https://archon-studio.com/files/manuals/homm/HoMM-Stronghold-Mission-Book-Beta_EN.pdf}{Corrosion} token. That token is removed, when that Unit takes damage.

Reviewer: ```suggestion
  \item When a Unit with \svg{defense} is attacked but suffers no \svg{damage-red}, it gains a Corrosion token. That token is removed when that Unit takes damage.
```
This URL is bound to fall out of date. Maybe let's leave it as it is?

  \item Additional Trading Post actions:
  \begin{itemize}

>     \item You may Search (2) \svg{artifact} and \svg{hand} 6 \svg{gold} for minor, 8 for \svg{gold} or 11 for \svg{gold} for Relic.

Reviewer: ```suggestion
    \item You may Search (2) \svg{artifact} and \svg{pay_v2} 6 \svg{gold} for minor, 8 \svg{gold} for major, or 11 \svg{gold} for Relic.
```

>     \item For 2×\textbf{x} +2  \svg{gold}, you may remove \textbf{any} Card from your hand or discard (\textbf{x} is the amount of times you have used this mode).

Reviewer: ```suggestion
    \item For $2×X 2$ \svg{gold}, you may remove \textbf{any} Card from your hand or discard (\textbf{x} is the amount of times you have used this mode).
```
Maybe math mode will work better?

  \end{itemize}
  \item Split the \svg{artifact} and Spell Decks as depicted in the Tournament Book.

>   \item In a 2-Player game when a Player gains \textbf{x} VP for controlling the Hill. That Player also has to distribute \textbf{x} \svg{damage-red} to their Units.

Reviewer: ```suggestion
  \item In a 2-Player game when a player gains \textbf{x} VP for controlling the Hill, that player also has to distribute \textbf{x} \svg{damage-red} to their Units.
```
What happens to this damage? Is it retained until the next combat starts? What if you assign it to units which won't be deployed to this combat?

Author: I think it is supposed to last until the next combat.

Author: hmm yeah, not sure. (this is Kablaamio's scenario and he said this was playtested, so maybe just rewording could be done)

Author: true

>   \item In a 2-Player game when a player gains \textbf{x} VP for controlling the Hill, that player also has to distribute \textbf{x} \svg{damage-red} to their Units. The \svg{damage-red} is retained until that Unit's next Combat.

Reviewer: Can you kill a unit with that damage?

Author: yeah. For example you could put the damage to your 0 cost gremlin and then buy it back before next combat. Too me it would be trivial that, that unit then dies. Even tho it did not receive the damage during combat

Reviewer: Trivial, but this rule doesn't state what happens exactly. If the damage is retained until next combat, do you keep zombie gremlins in your army, and deploy them with 0 hp to instantly die? What if you have a reinforced unit with 3 hp, and you assign 3 damage to it?

Author: I don't know how else to interpret it other than "When a When a Reinforced Reinforced unit takes damage equal to  unit takes damage equal to or greater than its maximum HP, flip it over to or greater than its maximum HP, flip it over to the “Few” side." and if a unit on the few side gets its HP to 0, it is dead. I know the rule is what happens inside combat board, but only way I could think of doing this is sam way what happens to units in combat, they die. So I don't see there being a zombie gremlin because you just don't have it anymore.

I don't know how how it could be clarified more 🤔

Reviewer: All right then, all this can be considered common sense. I'm still not sure if you can assign, say, 4 dmg to a 2 hp unit like gremlins. Is that a legal move?

Author: Yeah, that was what I got in mind yesterday too. I changed it so you have to always just give 1 dmg to your highest cost unit.

% !TeX spellcheck = en_US
\addscenariosection{1}{Clash Scenario}{King of the Hill v2}{\images/kingofthehill.png}

\begin{multicols}{2}

\textbf{Author:} KABLAMIO \& Cantila

\textbf{Source:} \href{https://discord.com/channels/1275490124301467799/1284428969369665631}{KABLAMIO's Discord}

> \textit{Win (and lose) PvP fights and hold the Hill.}

Reviewer: This could use some lore text 😆

Author: Yeah, I could come up with something short soon.

\subsection*{\MakeUppercase{Scenario Length}}
This Scenario plays out over 6 Rounds.

\subsection*{\MakeUppercase{Player Setup}}
\textbf{Player Count:} 2 -- 3

\textbf{Starting Resources:} 22 \svg{gold}, 5 \svg{building_materials}

\textbf{Starting Income:} 15 \svg{gold}, 2 \svg{building_materials}, 1 \svg{valuables}

\textbf{Starting Units:}
\begin{itemize}

>   \item A Few of \silver\ with the lowest Recruitment cost.

Reviewer: ```suggestion
  \item A Few of cheapest \silver\ Units
```

\end{itemize}

\textbf{Town Buildings:}
\begin{itemize}
  \item \bronze\ Dwelling
\end{itemize}

\subsection*{\MakeUppercase{Map Setup}}
Take the following Map Tiles and arrange them as shown in the Scenario map layout ($P$ stands for the number of players):

\begin{itemize}
  \item P × Starting (I) Map Tile
  \item P × Far (II-III) Map Tiles
  \item P × Near (IV-V) Map Tiles
  \item Center Map Tile (C1 or C3)
  \item Custom Center Map Tile
\end{itemize}

> \subsection*{\MakeUppercase{Victory Conditions}}
> The game ends at the end of the Round 6.
> 

Reviewer: ```suggestion
```

Same here, I guess it's redundant.

\subsection*{\MakeUppercase{Victory Points}}
After Round 6, the player with the most Victory Points (VP) wins.
Players get VPs for:
\begin{itemize}
  \item $R-1$ VPs for starting your turn on ``The Hill'', where $R$ is Round number.
  \item 2 VP for winning a Combat against another Player
  \item 1 VP for losing a Combat against another Player
\end{itemize}

\subsection*{\MakeUppercase{Timed Events}}

\textbf{\nth{2} Round:}
\begin{itemize}
  \item Gain Secondary Hero for free.
\end{itemize}
\textbf{\nth{3} Round:}
\begin{itemize}
  \item Gain Citadel for free. If you already had it, gain 1 VP.
\end{itemize}
\textbf{\nth{4} Round:}
\begin{itemize}
  \item Gain \silver\ dwelling for free. If you already had it, gain 2 VP.
\end{itemize}
\textbf{\nth{5} Round:}
\begin{itemize}
  \item Gain \golden\ dwelling for free. If you already had it, gain 3 VP.
\end{itemize}

\subsection*{\MakeUppercase{Additional Rules}}
\begin{itemize}
  \item Players also gain income on even-numbered Rounds.
  \item Players can't attack their opponent's starting Town.
  \item Players can only attack another player once each turn.
  \item Players do not lose gold for losing or retreating from any Combat.

>   \item Secondary Heroes act as Main Hero.

Reviewer: ```suggestion
\item Secondary Heroes act as Main Heroes, but keep their 2 \svgeven{movement}.
```
Maybe like that (if that's the intention)?

Author: they are supposed to have also 3 MPs. So it could be thought as your main hero is in two places at once. Or as in there is a clone of your main hero.

>   \item Secondary Heroes act as Main Hero.

Reviewer: ```suggestion
  \item Secondary Heroes act as Main Heroes, with full deck access and 3 \svgeven{movement}.
```
Maybe like that? How about empowerment effects? Do they gain experience?

Author: yes, they can also gain experience
